### ISSUE TITLE
Renaming sequence in the chain has an issue.

### ISSUE THREAD

**Issue by ohdongha** [hillerlab/make_lastz_chains#8]
Hi again,
I saw the update to deal with the sequence IDs with "." (per issue #3):

> Warning If your scaffold names are numbered, such as NC_00000.1, please consider removing scaffold numbers (rename NC_00000.1 to NC_00000 or NC_00000__1, for example). ... The pipeline will try to trim scaffold numbers automatically to process the data correctly. Afterward, it will rename the scaffolds back.

Since it appears that you implemented a mechanism to deal with the ".*" in sequence IDs, I downloaded the new version of the pipeline and ran it with the input genomes without modifying sequence IDs.   

The pipeline started with creating a renamed `.fasta` file and a `.tsv` that contains the sequence ID conversion table.  And at the end, the log says the pipeline is renaming chromosome names in the chain file.

However, the resulting chain file does not look right. Most of all, the chain score field is gone, making it impossible to salvage the results:
```
$ zgrep "chain" h38w5.m39w5.allfilled.chain.gz | head -n5 | column -t
chain  NC_000014.9   NC_000014  107043718  +  24687985   NC_000078.7  NC_000078  120092757  +  44683557   114519917  1
chain  NC_000004.12  NC_000004  190214555  +  1103677    NC_000071.7  NC_000071  151758149  +  33338229   105126044  2
chain  NC_000002.12  NC_000002  242193529  +  96496148   NC_000067.7  NC_000067  195154279  +  36305135   93990572   3
chain  NC_000001.11  NC_000001  248956422  +  930892     NC_000070.7  NC_000070  156860686  -  516431     53690008   4
chain  NC_000001.11  NC_000001  248956422  +  68121445   NC_000069.7  NC_000069  159745316  -  100601     72842298   5
``` 

For comparison, below is the chain file using a previous version of `make_lastz_chains`.  Here, I supplied the input genomes with sequence IDs pre-modified: 
```
$ zgrep "chain" h38w4.m39w4.all.chain.gz | head -n5 | column -t 
chain  720570302  NC_000014__9   107043718  +  24687985   106824309  NC_000078__7  120092757  +  44683557   114419863  1
chain  598452391  NC_000004__12  190214555  +  1104636    88140396   NC_000071__7  151758149  +  33339160   105126044  2
chain  584782083  NC_000001__11  248956422  +  930892     58547094   NC_000070__7  156860686  -  516431     53690008   3
chain  578674369  NC_000002__12  242193529  +  96496148   241869966  NC_000067__7  195154279  +  36305135   93989195   4
chain  564776354  NC_000001__11  248956422  +  68121445   158184951  NC_000069__7  159745316  -  100601     72842298   5
```

Is there any other part of the pipeline updated since July 6th, 2022?  If not, we will revert back to the older version (make_lastz_chains) and use genome files with the modified sequence IDs.   The current version has an issue. 

Thanks, 
Dong-Ha

**Comment by kirilenkobm** [comment]
Hi Dong-Ha,

I am sorry for that, just fixed the issue (I assigned updated chrome names to the wrong fields...) https://github.com/hillerlab/make_lastz_chains/commit/9cd91830be58e3a9a9f94a6a91abd02964355f9d

Now they are correct.

- Bogdan

**Comment by ohdongha** [comment]
Thanks, Bogdan, for the quick response.  

Is there anything else updated since July 6th, 2022, other than the renaming of sequence IDs? 

 Dong-Ha

**Comment by ohdongha** [comment]
Thanks, Bogdan, for responding quickly.  I have already submitted a round of runs using genomes renamed ("." -> "__") manually.  I will test this fix in the next round and close the ticket when it goes through. 

Dong-Ha

**Comment by ohdongha** [comment]
I tested the script downloaded on Oct. 10th (before the fix for #10 but after https://github.com/hillerlab/make_lastz_chains/commit/9cd91830be58e3a9a9f94a6a91abd02964355f9d), and it worked well with sequence IDs that contain "." and generated the correct `chain.gz` file.

Thanks!
Dong-Ha
